### PR TITLE
[ownership] cd7c9e97e46a0141e37f10fbfb74c094e0576ff3 followups

### PR DESCRIPTION
**Pull request by gottesmm**
Fixes stuff from #35522.

### CODE REVIEW

**Comment by gottesmm** [comment]
@swift-ci smoke test

**Comment by gottesmm** [comment]
@swift-ci test windows platform

**Comment by gottesmm** [comment]
@swift-ci smoke test

File: lib/SILOptimizer/SILCombiner/SILCombinerCastVisitors.cpp [hunk 1]
```diff
@@ -581,33 +587,67 @@ SILInstruction *SILCombiner::visitUnconditionalCheckedCastAddrInst(
...
   SILValue val = getConcreteValueOfExistentialBoxAddr(uccai->getSrc(), uccai);
   while (auto *cvi = dyn_cast_or_null<CopyValueInst>(val))
     val = cvi->getOperand();
   if (canBeUsedAsCastDestination(val, uccai, DA)) {
     // We need to copy the value at its insertion point.
     {
-      auto *valInsertPt = val->getDefiningInsertionPoint();
-      if (!valInsertPt)
+      auto *nextInsertPt = val->getNextInstruction();
+      if (!nextInsertPt)
```
Comment (atrick) [inline lib/SILOptimizer/SILCombiner/SILCombinerCastVisitors.cpp:635]: I actually don't think we should ever bailout after `getNextInstruction`. Can't we just assert in side `getNextInstruction` that it isn't called on a terminator?

**Review by atrick — COMMENTED** [review]
@gottesmm thanks. I have one post-commit review comment.

**Pull request merged**